shared/src/main/scala/squants/information/DataRate.scala

```diff
@@ -32,6 +32,25 @@ final class DataRate private(val value: Double, val unit: DataRateUnit)
   def toExbibytesPerSecond = to(ExbibytesPerSecond)
   def toZebibytesPerSecond = to(ZebibytesPerSecond)
   def toYobibytesPerSecond = to(YobibytesPerSecond)
+
+  def toBitsPerSecond = to(BitsPerSecond)
+  def toKilobitsPerSecond = to(KilobitsPerSecond)
+  def toMegabitsPerSecond = to(MegabitsPerSecond)
+  def toGigabitsPerSecond = to(GigabitsPerSecond)
+  def toTerabitsPerSecond = to(TerabitsPerSecond)
+  def toPetabitsPerSecond = to(PetabitsPerSecond)
+  def toExabitsPerSecond = to(ExabitsPerSecond)
+  def toZettabitsPerSecond = to(ZettabitsPerSecond)
+  def toYottabitsPerSecond = to(YottabitsPerSecond)
+
+  def toKibibitsPerSecond = to(KibibitsPerSecond)
+  def toMebibitsPerSecond = to(MebibitsPerSecond)
+  def toGibibitsPerSecond = to(GibibitsPerSecond)
```

Comment:
These are new to me. Do you have a reference to cite about these units?

Reply:
Concerning those IEC based units -  as far as i know they are rarely used. Still due to the fact that they were mentioned in https://en.wikipedia.org/wiki/Bit_rate and to be complementary with the added units for Information I decided to include them as well.